api/protobuf/buf.gen.yaml

```diff
@@ -0,0 +1,24 @@
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    # todo: re-organize this, keeping it to the old reference so consumers don't break
+    default: github.com/onflow/archive/api/archive
+    except:
+      - buf.build/googleapis/googleapis
```

Comment:
whatever dependencies you use, you would need to add them here.

Reply:
Done! :)